core/src/network/entry_with_header.rs

```diff
@@ -6,14 +6,14 @@ use std::{convert::TryInto, sync::Arc};
 
 #[derive(Serialize, Deserialize)]
 pub struct EntryWithHeader {
-    pub entry_body: Entry,
+    pub entry: Entry,
     pub header: ChainHeader,
 }
 
 impl EntryWithHeader {
     pub fn new(entry: Entry, header: ChainHeader) -> EntryWithHeader {
         EntryWithHeader {
-            entry_body: entry,
+            entry: entry,
```

Comment:
`entry: entry` -> `entry`

Reply:
Hey @sphinxc0re, please don't block for small single changes like this but instead use Github's suggestions feature:
```suggestion
            entry,
```